### ISSUE TITLE
Machine output set to configurable modID List

### ISSUE THREAD

**Issue by bookerthegeek** [Railcraft/Railcraft#1750]
# Heyo
So hello again! Love the mod and what you are all doing with it. Anywise.... Here I am tweaking my mod-pack and unifying everything because why not right? and I came across a couple of other mods that added in a config that I would love for Railcraft to have. The example below is from Nuclearcraft.

```
    # Determines the priority of each mod's ore dictionary entries - earlier mods' IDs in this list have a higher priority. This is used to determine which mod's corresponding item is produced in machine recipes. If no matching mod is found, the default entry is used.
    S:ore_dict_priority <
        railcraft
        techreborn
        nuclearcraft
        immersiveengineering
        minecraft
        thermalfoundation
        mekanism
        appliedenergistics2
     >
```

### Alternatives
So I've tried a couple different things.... Modtweaker was my goto and works wonders, let me tell you. Sadly it does not work for some machines. Mainly the ones from COFH and Railcraft, although due to removing some things due to overlap most of the offending COFH machines have just been turned off.

So, now every machine for example gives the TechReborn copper plate except for the rolling machine which gives me a thermal expansion plate. Which would be fine except I've changed every recipe to take the TechReborn plates. I could add a recipe to convert `<ore:plateCopper>` to `<techreborn:plates:20>` but an easier way (For me, not you) would be to have an option for the default mod owner as a config option.

This is by no means the only use case (Rolling Machine, Coke Oven, Blast Furnace, Rock Crusher, Steam Oven, Metals Chest) but just the easiest one to explain the situation.

### Thank ye fer all yer consideration
Thanks for reading this issue and I look forward to your response and hearing your opinions on this.

**Comment by liach** [comment]
https://github.com/Railcraft/Railcraft/issues/1637#issuecomment-453805546

> That's how you end up with dog poo cookies. Just because dog poo and cocoa are both "dyeBrown", doesn't mean you can make cookies with either.

I think the issue can be merged into #1610 so that players can customize all railcraft recipes with some other mods.

**Comment by liach** [comment]
Moving to #1610 as booker agreed.
